frontend/src/components/new-sightings-page/NewSightingForm.tsx

```diff
@@ -67,6 +67,23 @@ export const NewSightingForm: React.FC<NewSightingFormProps> = ({
 
   const handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
     setLocationInputType(event.target.value as LocationInputType);
+    setFormValues({
+      ...formValues,
+      longitude: "",
+      latitude: "",
+    });
+  };
+
+  const getLocationFromPhone = () => {
+    function success(pos: GeolocationPosition) {
```

Comment:
I think a better name is needed - "success" is a noun (object word), but functions are usually best named as verbs (doing words). If this is a callback, should it be `onSuccess`?
```suggestion
    function onSuccess(pos: GeolocationPosition) {
```